{"name": "test_login_locked_out_user", "status": "passed", "steps": [{"name": "Открываем страницу логина", "status": "passed", "start": 1758715859780, "stop": 1758715860799}, {"name": "Логинимся пользователем 'locked_out_user'", "status": "passed", "parameters": [{"name": "username", "value": "'locked_out_user'"}, {"name": "password", "value": "'secret_sauce'"}], "start": 1758715860799, "stop": 1758715860869}, {"name": "Получаем текст ошибки при логине", "status": "passed", "start": 1758715860869, "stop": 1758715860875}], "start": 1758715859780, "stop": 1758715860875, "uuid": "2d7c983c-e0e6-405f-b2c8-4912ba405d51", "historyId": "364d39b2be4aad43ae744509fff753ed", "testCaseId": "364d39b2be4aad43ae744509fff753ed", "fullName": "tests.ui.test_auth#test_login_locked_out_user", "labels": [{"name": "parentSuite", "value": "tests.ui"}, {"name": "suite", "value": "test_auth"}, {"name": "host", "value": "DESKTOP-JVPC402"}, {"name": "thread", "value": "14568-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.ui.test_auth"}], "titlePath": ["tests", "ui", "test_auth.py"]}